kvmagent/kvmagent/plugins/vm_local_volume_cache_plugin.py

```
@@ -0,0 +1,1173 @@
+
+        for lv_object in lv_objects:
+            lv_tags = lv_object[LVInfoFields.LV_TAGS.value].split(",")
+            pool_uuid_tags = list(filter(lambda tag: tag.startswith(cls.VM_LOCAL_VOLUME_CACHE_POOL_UUID_LVM_TAG_PREFIX),
+                                         lv_tags))
+            mount_path_tags = list(filter(lambda tag: tag.startswith(cls.VM_LOCAL_VOLUME_CACHE_POOL_MOUNT_PATH_LVM_TAG_PREFIX),
+                                         lv_tags))
+            assert pool_uuid_tags, "LV %s in VG %s is tagged with %s but missing pool UUID tag, skip loading this LV as a cache pool" % (
+                lv_object[LVInfoFields.LV_NAME.value], lv_object[VGInfoFields.VG_NAME.value], cls.VM_LOCAL_VOLUME_CACHE_POOL_MANAGED_LVM_TAG)
+            assert mount_path_tags, "LV %s in VG %s is tagged with %s but missing mount path tag, skip loading this LV as a cache pool" % (
+                lv_object[LVInfoFields.LV_NAME.value], lv_object[VGInfoFields.VG_NAME.value], cls.VM_LOCAL_VOLUME_CACHE_POOL_MANAGED_LVM_TAG)
+            pool_uuid = pool_uuid_tags.pop().strip(cls.VM_LOCAL_VOLUME_CACHE_POOL_UUID_LVM_TAG_PREFIX).strip("::")
```

Ruff (0.15.2)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)